benchmarks/bench_predictor.py

```diff
@@ -1,46 +1,75 @@
 from time import time
+
 import numpy as np
 from numpy.testing import assert_allclose
+from sklearn.datasets import make_regression
 from pygbm.grower import TreeGrower
+from pygbm.binning import BinMapper
+from pygbm.loss import LeastSquares
 
 
 # TODO: make some random numerical data, bin it, and fit a grower with many
 # leaves in a joblib'ed function.
 
+n_samples = int(5e6)
+
+X, y = make_regression(n_samples=n_samples, n_features=5)
+bin_mapper = BinMapper()
+X_binned = bin_mapper.fit_transform(X)
+
+loss = LeastSquares()
+gradients, hessians = loss.init_gradients_and_hessians(
+    n_samples, n_trees_per_iteration=1)
+y_pred_init = np.zeros_like(y)
+loss.update_gradients_and_hessians(gradients, hessians, y, y_pred_init)
 
-predictor = grower.make_predictor(bin_thresholds=bin_thresholds)
-binned_features_c = np.ascontiguousarray(binned_features)
+grower = TreeGrower(X_binned, gradients, hessians, max_leaf_nodes=None)
+grower.grow()
+
+predictor = grower.make_predictor(bin_thresholds=bin_mapper.bin_thresholds_)
```

Comment:
Rather than building the predictor from the low level API one could probably use the high level GradientBoostingRegressor API and then take:

```
predictor = regressor.predictors_[0]
```